backends/v2v/cfg/tests-shared.cfg

```diff
@@ -8,19 +8,19 @@ vm_type = v2v
 # The hypervisor uri (default, qemu://hostname/system, etc.)
 # where default or unset means derive from installed system
 connect_uri = default
-# Don't store vm info for v2v
-store_vm_info = no
 
 # Include the base config files.
 include base.cfg
-include subtests.cfg
 include machines.cfg
 include guest-os.cfg
 include guest-hw.cfg
 # Include configuration for v2v tests
 include convert_source.cfg
 include convert_destination.cfg
+include subtests.cfg
```

Comment:
why change the order 'include subtests.cfg' ? For v2v , I remember there is special requirement for include order

Reply:
The old implementation was by patch 'v2v-subtests.patch' in libvirt-ci. It's better to merge it.